[2022-01-29 12:26:39,974] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcs_dag.local_to_gcs_task scheduled__2019-01-07T00:00:00+00:00 [queued]>
[2022-01-29 12:26:40,192] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcs_dag.local_to_gcs_task scheduled__2019-01-07T00:00:00+00:00 [queued]>
[2022-01-29 12:26:40,192] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-01-29 12:26:40,192] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-01-29 12:26:40,192] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-01-29 12:26:40,386] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): local_to_gcs_task> on 2019-01-07 00:00:00+00:00
[2022-01-29 12:26:40,465] {standard_task_runner.py:52} INFO - Started process 3229 to run task
[2022-01-29 12:26:40,508] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion_gcs_dag', 'local_to_gcs_task', 'scheduled__2019-01-07T00:00:00+00:00', '--job-id', '232', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_gcs_dag.py', '--cfg-path', '/tmp/tmp3virzmmd', '--error-file', '/tmp/tmpu4w_tmmn']
[2022-01-29 12:26:40,523] {standard_task_runner.py:77} INFO - Job 232: Subtask local_to_gcs_task
[2022-01-29 12:26:40,973] {logging_mixin.py:109} INFO - Running <TaskInstance: data_ingestion_gcs_dag.local_to_gcs_task scheduled__2019-01-07T00:00:00+00:00 [running]> on host 85e1117cfb12
[2022-01-29 12:26:41,320] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=data_ingestion_gcs_dag
AIRFLOW_CTX_TASK_ID=local_to_gcs_task
AIRFLOW_CTX_EXECUTION_DATE=2019-01-07T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-01-07T00:00:00+00:00
[2022-01-29 12:26:42,808] {python.py:175} INFO - Done. Returned value was: None
[2022-01-29 12:26:43,372] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=data_ingestion_gcs_dag, task_id=local_to_gcs_task, execution_date=20190107T000000, start_date=20220129T122639, end_date=20220129T122643
[2022-01-29 12:26:43,589] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-01-29 12:26:44,023] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
